test/api/buffer/receivers/list/005-list-count-2/request.json

```diff
@@ -1,8 +1,7 @@
 {
   "path": "/v1/receivers",
   "method": "GET",
-  "body": {
-    "name": "Github Pull Requests",
-    "exports": []
```

Comment:
Typo

Reply:
jj, to som mazal z viacerych, a tu to zostalo